### PR TITLE
Currency conversion functionality added

### PR DESCRIPTION
**Pull request by tsayeau**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -132,7 +133,7 @@ bot.addListener('error', function(message) {
 bot.addListener("join", function(channel, who) {
 	// Welcome them in!
 	console.log("who: ", who);
-	// if(who.indexOf("panny")>-1) bot.say(channel, who+" "+randomFromArray(curlsForDaGirls));
+	if(who.indexOf("danranges")>-1) bot.say(channel, who+" lighten up");
```
Comment (chris30269) [inline main.js]: pls don't anger dan

File: main.js [hunk 2]
```diff
@@ -424,6 +425,16 @@ bot.addListener("message", function(from, to, text, message) {
 		}
 		else bot.say(config.channels[0], "But where tho");
 	}
+	if(splitup[0].toLowerCase() == "..$" || splitup[0].toLowerCase() == "!$" || splitup[0].toLowerCase() == ".$" || splitup[0].toLowerCase() == "..$" || splitup[0].toLowerCase() == "!$" || splitup[0].toLowerCase() == ".$"){
```
Comment (chris30269) [inline main.js]: lol pls

**Comment by chris30269** [comment]
Also, please add to the .help for this function

File: main.js [hunk 3]
```diff
@@ -511,4 +522,25 @@ function getWeather (where) {
 	    }
 	  }
 	});
+}
+function getCurrency (where) {
+	var appid = "c7ff9f2a57486218da1f";
+	var url = "https://free.currencyconverterapi.com/api/v6/convert?q="+currency+"_USD&compact=ultra&apiKey="+appid;
```
Comment (chris30269) [inline main.js]: define a variable before you use it maybe

Comment (chris30269) [reply main.js]: the perils of copy/paste

**Comment by chris30269** [comment]
Can you make it do the math?

`.$ 100 nok`

would spit out

`8.78 FREEDOM DOLLARS`

File: main.js [hunk 4]
```diff
@@ -525,7 +525,7 @@ function getWeather (where) {
 }
 function getCurrency (where) {
 	var appid = "c7ff9f2a57486218da1f";
-	var url = "https://free.currencyconverterapi.com/api/v6/convert?q="+currency+"_USD&compact=ultra&apiKey="+appid;
+	var url = "https://free.currencyconverterapi.com/api/v6/convert?q="+where+"_USD&compact=ultra&apiKey="+appid;
```
Comment (chris30269) [inline main.js]: HAHAHA

Comment (tsayeau) [reply main.js]: its fine

Comment (tsayeau) [reply main.js]: it works

**Pull request merged**